### ISSUE TITLE
Thats impressive work!!!! well done!

### ISSUE THREAD

**Issue by nixinator** [hugolgst/nixos-raspberry-pi-cluster#1]
I'll be trying this out soon, with the PI 4's, i wonder what ipxe network booting of these could do wtih nixos too?

It's may be possible!

****Nice stuff!****

**Comment by hugolgst** [comment]
I am not sure to understand the question :/

Otherwise, thanks a lot!

**Comment by nixinator** [comment]
https://github.com/ipxe/pipxe

**Comment by JustAlternate** [comment]
I agree, this repository is gold for me too. Thanks for sharing it out on github

Im in the process of buying different Raspberry Pi to make a cluster using only Nix

Cheers from Nantes :fr:

**Comment by hugolgst** [comment]
Bonne chance! 

Let me know if there's anything I can do to help
